src/Sarif.Viewer.VisualStudio/Themes/DefaultStyles.xaml

```diff
@@ -49,6 +49,23 @@
                 Value="Courier New" />
         <Setter Property="FontSize"
                 Value="12" />
+        <Style.Triggers>
+            <DataTrigger Binding="{Binding Location.Importance}"
+                         Value="Essential">
+                <Setter Property="FontWeight"
+                        Value="Bold" />
```

Comment:
Huh, for some reason I never think of using style triggers for things like this. I would've written an `ImportanceToFontWeightConverter`. Your way is much better.